continuous-integration/devcontainers/terraform/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM golang:1.13-stretch
```

Comment:
Do we want a step in the CI to ensure that this and other containers in this library build properly?